### ISSUE TITLE
i can't open my Rstudio

### ISSUE THREAD

**Issue by armstrang** [rstudio/rstudio#2610]
### System details

    RStudio Edition : Desktop 
    RStudio Version : 1.1.423
    OS Version      : windows 10 1803
    R Version       : 3.4.3

### Describe the problem in detail
i can't open my Rstudio, but R works on the Jupyter Notebook
### Describe the behavior you expected

![untitled](https://user-images.githubusercontent.com/32195239/38456880-eaf5b48a-3abb-11e8-97f9-12ce81e5f98d.png)

![cajure](https://user-images.githubusercontent.com/32195239/38456779-f013825e-3aba-11e8-8302-00716f793d23.PNG)

**Comment by jmcphers** [comment]
Sorry, we probably won't be able to solve this without more detail and ideally some reproducible steps. Here's a good place to start with troubleshooting:

https://support.rstudio.com/hc/en-us/articles/200488508-RStudio-Desktop-Will-Not-Start

If that doesn't help, this forum isn't the right place (it's for reporting bugs in the product). Instead I'd suggest asking for help on the community forum:

https://community.rstudio.com/

Hope this helps!

**Comment by gjack-view** [comment]
I believe I've encountered the same issue. I updated two computers to WIndows10 build1803 and am running RStudio from the Anaconda distribution. I am able to run R Terminal, but RStudio goes to the Download File prompt shown by OP.  My other programs all appear to be running fine. 

I've tried deleting RStudio-Desktop appdata and uninstalling/reinstalling RStudio with no luck. Is there a compatibility issue with the latest Windows and R-Studio? Is there something I can select at the Download File prompt that might allow RStudio to load?
